manual/source/extensions/mps/designs.py

```diff
@@ -34,13 +34,22 @@
 
 '''
 
+# Macros that are improperly named (not all-caps).
```

Comment:
That's a good point.  These macros were inherited from work by Tony and don't fit MPS conventions.  Logged in #122